runtime/queries/htmldjango/textobjects.scm

```diff
@@ -0,0 +1,7 @@
+[
+  "{{"
+  "}}"
+  "{%"
+  "%}"
+  (end_paired_statement)
+] @tag
```

Comment:
I'm not quite sure which textobject this should be. Maybe `@function.around`? Or it could use the ones from https://github.com/helix-editor/helix/pull/3282?

Reply:
I am not sure either, but since I probably won't use textobjects.scm anytime soon so I guess I will just remove it.